test/generators/pry_byebug_install_generator_test.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "generators/boring/letter_opener/install/install_generator"
+
+class AuditInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::PryByebug::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+
+  def test_should_install_pry_byebug_gem
+    Dir.chdir(app_path) do
+      quietly { generator.add_pry_byebug_gem }
```

Comment:
We can just run the whole generator with `run_generator` instead of generator.add.... since there are no additional configurations or methods in the generator.

We use "generator.add...." because running the whole generator takes too much time when there are other configurations. It saves time to just test what is intended so we avoid running whole generator unless we absolutely need to for testing.